### PR TITLE
Update temporary specs

### PR DESCRIPTION
**Pull request by kachick**
* Remove when some specs exist
* Comment-in

### CODE REVIEW

File: library/net/ftp/set_socket_spec.rb [hunk 1]
```diff
@@ -4,5 +4,5 @@
 describe "Net::FTP#set_socket" do
   # TODO: I won't spec this method, as it is not used
   #       anywhere and it should be private anyway.
-  #it "needs to be reviewed for spec completeness"
+  it "needs to be reviewed for spec completeness"
 end
```
Comment (eregon) [inline library/net/ftp/set_socket_spec.rb]: I would just remove the file entirely.
The comment seems wrong, that method seems public in MRI 2.6 at least.

**Comment by eregon** [comment]
I wonder if we should remove the rest too, at least those are not counted in number of examples.
It's useful to have the boilerplate generated by `mkspec` though.
So probably fine as-is for the rest.

File: library/net/ftp/set_socket_spec.rb (hunk 1 above)
Comment (eregon) [reply library/net/ftp/set_socket_spec.rb]: Ah it's fine, the comment says `it should be private`, not that it is already private.

**Comment by kachick** [comment]
Thank you!

**Pull request merged**